services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/entity/UTL3NeighborInfoMapping.java

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.dataplane.entity;
+
+import java.util.List;
+
+public class UTL3NeighborInfoMapping {
```

Comment:
Is this possible to move three new UT classes to under /test as they are pure test classes?

Reply:
Sure